tests/Test/AbstractFixerTestCase.php

```diff
@@ -155,8 +155,20 @@ final public function testFixerDefinitions(): void
                 $configSamplesProvided['default'] = true;
             }
 
-            if ($sample instanceof VersionSpecificCodeSampleInterface && !$sample->isSuitableFor(\PHP_VERSION_ID)) {
-                continue;
+            if ($sample instanceof VersionSpecificCodeSampleInterface) {
+                $supportedPhpVersions = [7_04_00, 8_00_00, 8_01_00, 8_02_00];
```

Comment:
Is it possible to get this collection of supported versions through some common service? We could use it on other places that operate on versions, like `if (\PHP_VERSION_ID < 70400 || \PHP_VERSION_ID >= 80300) {` in main `php-cs-fixer` binary (well, it would require some trick for determining max version). Not a blocker here, just a thought that can be implemented later.

Reply:
That `if` condition in `php-cs-fixer` is before Composer's autoloader is run and I guess we don't want to change that.